nemesis/arena_battles.py

```diff
@@ -27,7 +27,7 @@ def save_game(first_save=False):
         pass
     while not nemesis.nemesis_pathing.set_movement([-2, -2]):
         pass
-    print("Arena - Touch Save Sphere, and actually save")
+    logger.debug("Arena - Touch Save Sphere, and actually save")
```

Comment:
I did not suggest that we go through all of nemesis to update all of the prints to logging. I only suggested we update the newly added prints to use logging.